main.go

```diff
@@ -1,11 +1,26 @@
 package main
 
 import (
+	"log"
+	"os"
+
 	"github.com/ivanterekh/go-skeleton/server"
 	"github.com/xlab/closer"
 )
 
 func main() {
-	server.Start(":8080")
+	env := os.Getenv("ENV")
+	if env == "" {
+		env = "unknown"
+	}
+	log.Printf("running app in %s environment", env) // TODO: change logging
+
+	port := os.Getenv("PORT")
```

Comment:
I'd prefer a default value instead of Fatal

Reply:
Done.